ci-scripts/yocto.groovy

```diff
@@ -131,6 +131,27 @@ void runSmokeTests(String yoctoDir, String imageName) {
     }
 }
 
+void runBitbakeTests(String yoctoDir) {
+    String archiveDir = "bitbake_TestReport"
+
+    try {
+        stage("Perform Bitbake Testing"){
+            vagrant("/vagrant/cookbook/yocto/run-bitbake-tests.sh")
+        }
+    } catch (e){
+	   echo "Bitbake tests failed"
+	   println(e.getMessage())
+    } finally{
+        stage("Publish bitbake test results") {
+            reportsDir="/vagrant/${archiveDir}/test_reports/bitbake_tests/"
+            vagrant("mkdir -p ${reportsDir}")
+            vagrant("cp -a ${yoctoDir}/build/TestResults* ${reportsDir}")
```

Comment:
This "TestResults" is something which is created during the smoke tests. I'm not sure this will be a success, unless the bitbake tests also generate this report with the same name. It might be that we instead want to copy a log unless we can find a result file.